roles/alertmanager/tasks/configure.yml

```diff
@@ -17,6 +17,7 @@
     group: alertmanager
     mode: 0644
     validate: "{{ _alertmanager_binary_install_dir }}/amtool check-config %s"
+  no_log: "{{ 'true' if __testing is defined else 'false' }}"
```

Comment:
Maybe I miss something, but shouldn't this be the opposite?

`'false' if __testing is defined else 'true'`

Reply:
Yes, that logic was backwards. Fixed.